index.js

```diff
@@ -39,14 +40,35 @@ module.exports = app => {
   app.on(['pull_request.opened', 'pull_request.reopened', 'pull_request.synchronized', 'pull_request.edited'], async context => {
     var pullRequest = context.payload.pull_request
 
-    var ghLabel = size(pullRequest.additions + pullRequest.deletions)
+    var additions = pullRequest.additions
```

Comment:
Can simplify some of this with [Destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)...

```js
const pullRequest = context.payload.pull_request;
const {owner: {login: owner}, name: repo} = pullRequest.base.repo;
const {number} = pullRequest;
let {additions, deletions} = pullRequest;

```